### PR TITLE
Fix Expand / Contract of the "Show" button

### PR DESCRIPTION
**Pull request by reyronald**
**Release Type:** *Non Breaking Feature*

Fixes https://x-team-internal.atlassian.net/browse/XP-2294
(Tabular view: Expand / Contract of the “Show” button when switching views)

## Description

Now, when switching between List and Tabular view,
the "Show" button will not expand, contract, move,
or show any animation. Basically the button animation
was disabled for this scenario.

Also, making the height of the button consistent
among the two different views

## Checklist

- [x] set yourself as an assignee
- [x] set appropriate labels for a PR (initially it's mostly `[zube]: In Review`)
- [x] set `[zube]: In Review` label for respective issue as well
- [x] make sure your code lints (`npm run lint`)
- [x] Flow typechecks passed (`npm run typecheck`)
- [x] my change requires a change to the component's documentation (`.snap`)
- [x] component's documentation (`.snap`) is changed or added accordingly
- [x] if you've fixed a bug or added code that should be tested, add unit tests
- [x] if any snapshots have been changed, verify that component still works and looks as expected and update changed snapshot
- [x] **manually tested the component** by running it in the browser and checked nothing is broken and operates as expected!

## Related PRs

N/A

## Steps to Test or Reproduce

Switch between List and Tabular view. The "Show" button should not jerk in position or change size with an animation.

## Impacted Areas in Application

* `SearchForm` component in Tabular and List view

## Screenshots

Before: https://www.useloom.com/share/ed18ec4fdf2940cabc1b55fa3c39f598

After:

xp-2294 b

### CODE REVIEW

**Comment by reyronald** [comment]
Merged and tested successfully in Chrome, Firefox and Edge

File: src/components/ui/SearchForm.js [hunk 1]
```diff
@@ -145,6 +146,7 @@ const tabularTheme = {
     margin: 0 10px
     height: 58px
     padding: 10px 40px
+    transition: margin, padding 0s
```
Comment (nicksp) [inline src/components/ui/SearchForm.js]: When we use `0` we don't need to specify any units. It's just useless 😄

**Comment by reyronald** [comment]
@nicksp That's because the `s` unit is necessary fir the `transition` property to work 😄 

before

Fixed:

image

I'm going to add the unit back

**Comment by reyronald** [comment]
Check the docs here: https://developer.mozilla.org/en-US/docs/Web/CSS/transition-delay#Formal_syntax

File: src/components/ui/SearchForm.js (hunk 1 above)
Comment (reyronald) [reply src/components/ui/SearchForm.js]: That's the case for size units, but not for time units 😅 . Check my comment here: https://github.com/x-team/auto-ui/pull/237#issuecomment-427159086

Removing the unit is what actually caused the bug that you saw

**Comment by nicksp** [comment]
@reyronald Ah, really, good to know. Actually, I recommend turning off transition which seems to do the same tricks but a bit more logical step.

**Pull request merged**